scss/overrides/_type.scss

```diff
@@ -0,0 +1,44 @@
+a {
+  text-decoration: none;
+}
+
+.learn-more {
```

Comment:
You're still seeing `.learn-more` here because that's what landed on master. `.learn-more` was removed from `scss/foundation-styles/type.scss` and put in `scss/overrides/_type.scss` in the footer pr(not landed yet) previous to this and can be seen [here](https://github.com/mozilla/network-pulse/pull/1054/files#diff-488978533a087768aae55850bb8b2b45L105). I'd rather clean this up in the footer pr or afterwards to avoid confusion or any weird merge conflicts if you're cool with that. Let me know which you prefer.

Reply:
Yup we can wait until the other PR is merged first. 

It's best to request PR reviews _only_ when the PR isn't blocked by other things as code reviewers review code against `master` and we don't have insights about the work in other PRs. 👍

Reply:
Cool, I'll flag this to go back and do this once that pr lands.